vllm/model_executor/models/aria.py

```diff
@@ -301,7 +303,9 @@ def __init__(
         prefix: str = "",
     ) -> None:
         super().__init__(config, cache_config, quant_config, prefix)
-        self.mlp = AriaTextMoELayer(config, quant_config=quant_config)
+        self.mlp = AriaTextMoELayer(config,
+                                    quant_config=quant_config,
+                                    prefix=f"{prefix}.moe")
```

Comment:
Think this needs to be mlp per https://huggingface.co/rhymes-ai/Aria?show_file_info=model.safetensors.index.json
```suggestion
                                    prefix=f"{prefix}.mlp")
```